src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -60,7 +60,12 @@ public sealed class Features
         /// Updates the FirstN/LastN functions to require a second argument, instead of
         /// defaulting to 1.
         /// </summary>
-        internal bool FirstLastNRequiresSecondArguments { get; set; }
+        internal bool FirstLastNRequiresSecondArguments { get; set; }
+
+        /// <summary>
+        /// Allow delegation for Min and Max tabular functions with Date and DateTime fields
+        /// </summary>
+        internal bool AllowDateTimeMinMaxDelegation { get; set; }
```

Comment:
This, and AllowImpure/AllowAsync delegation probably shouldn't be in Features?
They're PA specific, and I'd imagine belong more with the existing IExternal*Features